### DISCUSSION TITLE
lerna-schema.json has incorrect defintion for "contents" option (should be type: "string")

### DISCUSSION THREAD

**Discussion by chandu-ex** [ghiscoding/lerna-lite#459] [Q&A] [answered]
As per lerna documentation @ https://github.com/lerna/lerna/blob/main/commands/publish/README.md#--contents-dir the expected value is a directory name `(e.g: dist, build etc.)`
But in the lerna-schema.json definition of this @lerna-lite/cli package (https://github.com/lerna-lite/lerna-lite/blob/main/packages/cli/schemas/lerna-schema.json#L1626) the definition is marked as boolean
Shouldn't this be a string type i.e. 
```
"contents": {
  "type": "string",
  "description": "For `lerna bootstrap`, `lerna link`, and `lerna publish`, the subdirectory to use as the source of any links. Must apply to ALL packages."
}
```
?

**Comment by ghiscoding** [comment, marked as answer]
oh yeah you're right and I guess we could also remove the lerna bootstrap and link from the description since it doesn't exist in Lerna-Lite... would you like to do a PR to fix it?

**Reply by chandu-ex** [reply]
@ghiscoding  Sure, I will do that once I get to my personal laptop.

**Reply by ghiscoding** [reply]
turning this into an issue #466 to reference, I will go ahead with a PR #467 to fix this and also do a lot of cleanup in the Schema since I found out that there's a lot of properties that aren't included in Lerna-Lite and should be removed (ie: lerna `add`, `create`, `link` should all be removed since they will never be part of Lerna-Lite)
